src/frontend/src/lib/styles/global/button.scss

```diff
@@ -135,37 +135,4 @@ button {
 			}
 		}
 	}
-
-	&.authenticate {
-		position: relative;
-		z-index: 0;
-	}
-
-	&.authenticate::before {
-		border-radius: 9999px;
-		position: absolute;
-		content: '';
-		top: 0;
-		right: 0;
-		bottom: 0;
-		left: 0;
-		z-index: -1;
-		background: radial-gradient(
-			49.28% 49.28% at 50% 100%,
-			var(--color-turquoise) 0%,
```

Comment:
Colors can be cleaned in Tailwind config?

Reply:
done